### ISSUE TITLE
Method search/more_like_this

### ISSUE THREAD

**Issue by ianovi** [baileyherbert/envato.js#13]
Hi 🙂, thanks for the great lib!

Could you release subj method which described on https://build.envato.com/api/#search_getSearchMoreLikeThis ?
It looks that it should be simple. ;)
Or maybe i'll make update for you review?

Thanks in advance.

**Comment by baileyherbert** [comment]
Hi @HotTemplate! Thanks for letting me know that this endpoint was missing. I'm not sure how I missed it. 😅
I'll certainly add this feature for you as soon as I can. In the meantime, you can do it manually like so:

```ts
import { Client, Item } from 'envato';
import url from 'envato/dist/util/url.js';

const client = new Client('TOKEN');

const response = await client.get<IMoreLikeThisResponse>(
	url.build('/v1/discovery/search/search/more_like_this', {
		item_id: 17022701,
		page: 1,
		page_size: 100
	}
));

console.log(response.matches);

export interface IMoreLikeThisResponse {
	took: number;
	matches: Item[];
	item: Item;
}
```

**Comment by baileyherbert** [comment]
Sorry for the delay on this. I've added the missing endpoint and published it to npm as `envato@3.1.0` – cheers!

**Comment by ianovi** [comment]
Hi @baileyherbert, it seems (by the envato responses) in interface `IMoreLikeThisResponse` the `item` property should be `SimilarItem` instead `Item`.

**Comment by baileyherbert** [comment]
Thanks @HotTemplate! I've just released a fix as `envato@3.2.0`, cheers!
